java-frontend/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -582,39 +587,111 @@ private void executeMethodInvocation(MethodInvocationTree mit) {
       List<SymbolicValue> invocationArguments = invocationArguments(unstack.values);
       List<Type> invocationTypes = mit.arguments().stream().map(ExpressionTree::symbolType).collect(Collectors.toList());
 
-      methodInvokedBehavior.exceptionalPathYields()
-        .flatMap(yield -> yield.statesAfterInvocation(invocationArguments, invocationTypes, programState, () -> resultValue).stream())
-        .forEach(this::enqueueExceptionalPaths);
+      Map<Type, SymbolicValue.ExceptionalSymbolicValue> thrownExceptionsByExceptionType = new HashMap<>();
+
+      // Enqueue exceptional paths from exceptional yields
+      methodInvokedBehavior.exceptionalPathYields().flatMap(yield -> yield.statesAfterInvocation(
```

Comment:
please indent correctly and put new lines like the happyPathYields : flat map should be on next line.